lib/main.dart

```diff
@@ -19,16 +20,15 @@ $ex
 $stack''';
 }
 
-Future<void> reportError(dynamic ex, StackTrace stack) async {
-  if (Prefs.devMode.get()) {
+Future<void> reportError(dynamic error, StackTrace stack) async {
+  if (Prefs.isInitialized && Prefs.devMode.get()) {
     ShowFunctions.instance.showError(
-      errorDetails: buildErrorReport(ex, stack),
+      errorDetails: buildErrorReport(error, stack),
     );
   }
-  await FirebaseCrashlytics.instance.recordError(
-    ex,
-    stack,
-  );
+  if (Firebase.apps.isNotEmpty) {
+    await FirebaseCrashlytics.instance.recordError(error, stack, fatal: true);
```

Comment:
We need to mark errors as fatal now, otherwise they don't get reported ([docs](https://firebase.google.com/docs/crashlytics/customize-crash-reports?platform=flutter#report-caught-exceptions)):
![Firebase documentation for non-fatal errors](https://github.com/nt4f04uNd/sweyer/assets/6966049/0271741b-b292-4f9a-ae74-1cbced09c61a)

Reply:
Ok, but this is a weird move from them

Reply:
Agreed.